Neos.Flow/Classes/Mvc/Routing/Dto/MatchResult.php

```diff
@@ -31,14 +31,20 @@ final class MatchResult
      */
     private $tags;
 
+    /**
+     * @var RouteLifetime|null
+     */
+    private $lifetime;
+
     /**
      * @param mixed $matchedValue
      * @param RouteTags $tags
      */
-    public function __construct($matchedValue, RouteTags $tags = null)
+    public function __construct($matchedValue, RouteTags $tags = null, RouteLifetime $lifetime = null)
```

Comment:
nitpick: AFAIK we target PHP 8.1+ with Neos 8.3 so we could replace this with
```suggestion
    public function __construct(
        $matchedValue,
        RouteTags $tags = null,
        private readonly ?RouteLifetime $lifetime = null
    )
```

Reply:
To be in line with the rest of the file

Reply:
makes sense